libafl/src/events/llmp.rs

```diff
@@ -557,6 +594,27 @@ where
         Ok(())
     }
 
+    fn serialize_observers<OT>(&mut self, observers: &OT) -> Result<Option<Vec<u8>>, Error>
+    where
+        OT: ObserversTuple<Self::State> + Serialize,
+    {
+        if self.execution_time == Duration::ZERO
+            || self.serialization_time == Duration::ZERO
+            || self.serialization_time + self.deserialization_time < self.execution_time
+            || self.serializations_cnt.trailing_zeros() >= 8
```

Comment:
`self.serializations_cnt.trailing_zeros() >= 8`
what does this case mean?

Reply:
& 0xff, clippy doesn't like bitmasks